Library/Homebrew/language/node.rb

```diff
@@ -0,0 +1,35 @@
+module Language
+  module Node
+    def self.npm_cache_config
+      "cache=#{HOMEBREW_CACHE}/npm_cache\n"
+    end
+
+    def self.setup_npm_environment
+      npmrc = Pathname.new("#{ENV["HOME"]}/.npmrc")
+      # only run setup_npm_environment once per formula
+      return if npmrc.exist?
```

Comment:
You mean how likely it is that a project ships a `.npmrc` file inside a `.brew_home` folder in their tarball? I would say quite unlikely and even than we would probably run into other issues with `.brew_home` too.

BTW: It's quite funny how a fix for a unlikely edge case of having to install more than one node module per formula has lead to a discussion about an even less likely edge case here.